pkg/config/config_test.go

```diff
@@ -18,8 +18,8 @@ func TestLoadConf(t *testing.T) {
 	os.Setenv("NOTIFICATION_LOG_LEVEL", "debug")
 	os.Setenv("NOTIFICATION_GRPC_SHOW_ERROR_CAUSE", "false")
 
-	//os.Setenv("NOTIFICATION_MYSQL_HOST", "MYSQL_HOST_test")
-	//os.Setenv("NOTIFICATION_MYSQL_PORT", "8888")
+	os.Setenv("NOTIFICATION_MYSQL_HOST", "MYSQL_HOST_test")
+	os.Setenv("NOTIFICATION_MYSQL_PORT", "13306")
```

Comment:
Not string

Reply:
Setenv(key, value string) ，The Params should be string.